packages/core/test/helpers/exact-interface.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-unused-modules */
```

Comment:
I think we should try to remove all the eslint-disable directives later and see what happens.